cookbooks/aws-parallelcluster-entrypoints/kitchen.entrypoints-install.yml

```diff
@@ -0,0 +1,14 @@
+---
+verifier:
+  name: inspec
+  inspec_tests:
+    - test
+
+suites:
+  - name: setup_envars
+    run_list:
+      - recipe[aws-parallelcluster-common::setup_envars]
```

Comment:
why pointing to common? setup_envars is in the shared cookbook. 

then why defining these tests here in the entrypoints cookbook? I'd expect to have this test in the shared cookbook not here.